internal/controller/cluster_upgrade.go

```diff
@@ -704,7 +704,7 @@ func (r *ClusterReconciler) upgradeInstanceManager(
 				instanceManagerIsUpgrading,
 				operatorHash[:6],
 				instanceManagerHash[:6])
-			contextLogger.Trace(message)
+			contextLogger.Info(message)
```

Comment:
This would emit an info line for every pod on every reconciliation cycle because this block is executed when there is nothing to do.

Reply:
Then Trace XD